packages/fiber/src/core/hooks.tsx

```diff
@@ -125,14 +126,14 @@ export function useLoader<T, U extends string | string[], L extends LoaderProto<
   input: U,
   extensions?: Extensions<L>,
   onProgress?: (event: ProgressEvent<EventTarget>) => void,
-): U extends any[] ? BranchingReturn<R, GLTF, GLTF & ObjectMap>[] : BranchingReturn<R, GLTF, GLTF & ObjectMap> {
+): U extends any[] ? BranchingReturn<R, GLTFLike, R & ObjectMap>[] : BranchingReturn<R, GLTFLike, R & ObjectMap> {
   // Use suspense to load async assets
   const keys = (Array.isArray(input) ? input : [input]) as string[]
   const results = suspend(loadingFn<L>(extensions, onProgress), [Proto, ...keys], { equal: is.equ })
   // Return the object/s
   return (Array.isArray(input) ? results : results[0]) as U extends any[]
-    ? BranchingReturn<R, GLTF, GLTF & ObjectMap>[]
-    : BranchingReturn<R, GLTF, GLTF & ObjectMap>
+    ? BranchingReturn<R, GLTFLike, R & ObjectMap>[]
```

Comment:
Do we have any tests asserting the type output? Would like to see an output where we're not relying on type augmentation to access nodes / materials / etc.

Reply:
Looks like he shouldn't be declaring the generic?

Reply:
> Looks like he shouldn't be declaring the generic?

Probably, but I'll do integration tests to make sure this will "just work" when we release in R3F.

> Do we have any tests asserting the type output? Would like to see an output where we're not relying on type augmentation to access nodes / materials / etc.

Done in 803ee2c38ff727a8dbc0bd8044a43798eb9014df.